arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -143,7 +143,7 @@ static inline u32_t _get_region_index_by_type(u32_t type)
 }
 
 /**
- * This internal function check if region is enabled or not
+ * This internal function checks if region is enabled or not.
```

Comment:
Can you please make the same comment fixes in the nxp mpu?

Reply:
Will have a look now.